### ISSUE TITLE
Make Honeybee properties tab sticky

### ISSUE THREAD

**Issue by mostaphaRoudsari** [ladybug-tools/honeybee-rhino-plugin#114]
The honeybee info tab is very nice. Once I select an object and go to honeybee sub-tab I want it to stay there until I pick an object which is not a honeybee object. It looks like to be working fine if I select the same object type (e.g. select one zone after another) but if I pick a new object type like a shade then it will jump back to properties tab.

![Honeybee Rhino](https://user-images.githubusercontent.com/2915573/78503652-02f87c80-7736-11ea-863f-2b15fc22eab3.gif)

Is there a way to fix this behavior?

**Comment by MingboPeng** [comment]
This is because each type has its own property panel, Rhino only keeps the same property panel if it is the same as previous selected item.

I used to add all types into one property panel, which could achieve what you are looking for. But the reason I separated them into several independent panels is because there was a panel refresh issue when switching between different types. There was a significant refresh delay, that I have to unselect and select again to make it the panel render correctly.

However, I just tested again with everything on one panel, the refresh issue does not happen again. Rhino may have fixed this, or just because I am working on a desktop instead of my laptop I was working on. I will do more tests to ensure performance and user experience can both be achieved.
